pkgs/development/compilers/eunomia-bpf/eunomia-cc/default.nix

```diff
@@ -0,0 +1,96 @@
+{ lib
+, stdenv
+, makeWrapper
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, llvmPackages_16
+, elfutils
+, zlib
+}:
+let
+  bpftool = stdenv.mkDerivation {
+    pname = "bpftool";
+    version = "unstable-2023-03-11";
+    src = fetchFromGitHub {
+      owner = "eunomia-bpf";
+      repo = "bpftool";
+      rev = "05940344f5db18d0cb1bc1c42e628f132bc93123";
+      hash = "sha256-g2gjixfuGwVnFlqCMGLWVPbtKOSpQI+vZwIZciXFPTc=";
+      fetchSubmodules = true;
+    };
+
+    buildInputs = [ llvmPackages_16.clang llvmPackages_16.libllvm.dev elfutils zlib ];
+
+    buildPhase = ''
+      make -C src
+    '';
+
+    installPhase = ''
```

Comment:
Upstream's default `make install` doesn't work?

Reply:
> Upstream's default `make install` doesn't work?

This requires a particular output directory structure to fit the `build.rs` script, thus `make install` does not work, or will need more work.